leafletmap.js

```diff
@@ -1,26 +1,26 @@
 window.onload = function () {
     console.log("Script is running")
     var basemap = L.tileLayer('https://a.tile.openstreetmap.org/{z}/{x}/{y}.png', {
-		attribution: '&copy; <a href="https://osm.org/copyright">OpenStreetMap</a> contributors',
```

Comment:
The changes to HTML and JavaScript are not related to this pull request. Plus, the JavaScript changes break the indention by mixing tabs and spaces.

Reply:
Sorry, this actually removes the tabs and replaces it with spaces only. My mistake.
Still, this change has nothing to do with the topic of this pull request ;-)

Reply:
I removed the changes in the HTML and js file.